src/main/java/Main.java

```diff
@@ -9,7 +9,7 @@
 
 public class Main {
 
-    private static EmployeeFacade employeeFacade = new EmployeeFacadeImpl();
+    private static EmployeeFacade employeeFacade;
```

Comment:
this could be null because there is no instantiation of this class. please add it and test this properly